x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -21,30 +21,30 @@ import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
 import { PartitionElementEvent } from '@elastic/charts';
 import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
 import { CloudPostureScoreChart } from '../compliance_charts/cloud_posture_score_chart';
-import { useCloudPostureStatsApi } from '../../../common/api/use_cloud_posture_stats_api';
 import { ChartPanel } from '../../../components/chart_panel';
 import * as TEXT from '../translations';
-import { Evaluation } from '../../../../common/types';
+import { ComplianceDashboardData, Evaluation } from '../../../../common/types';
 import { RisksTable } from '../compliance_charts/risks_table';
 import { INTERNAL_FEATURE_FLAGS, RULE_FAILED } from '../../../../common/constants';
 import { useNavigateFindings } from '../../../common/hooks/use_navigate_findings';
 
-const logoMap: ReadonlyMap<string, EuiIconType> = new Map([['CIS Kubernetes', 'logoKubernetes']]);
+const logoMap: ReadonlyMap<string, EuiIconType> = new Map([
+  ['CIS Kubernetes V1.20', 'logoKubernetes'],
+]);
```

Comment:
any reason you favoured this over an object? (just curios)

Reply:
I just changed the text here a bit, If I'm not mistaken @ari-aviran changed it from an object to a set. might be wrong.
We can tackle this part in the future when we actually start working with a bunch of icons, for now, we are just displaying the k8s icon by default as asked by @kfirpeled